database/factories/OrderFactory.php

```diff
@@ -15,7 +15,7 @@ public function definition()
     {
         return [
             'user_id' => $this->faker->unique()->user id(),
-            'order_date' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
+            'order_date' => $faker->dateTime();
```

Comment:
This should be `$this->faker->dateTime()`. Did you try running the seeder to be sure there are no errors?